visual-novel-aztec/src/utils/ProgressTracker.ts

```
@@ -0,0 +1,102 @@
+    }
+    
+    if (this.progress.vocabularyLearned.length >= 10 &&
+        !this.progress.achievements.includes('language_learner')) {
+      this.progress.achievements.push('language_learner');  
+    }
+  }
+  
+  static getProgress(): GameProgress {
+    return this.progress;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)